pom.xml

```diff
@@ -10,11 +10,11 @@
 
     <inceptionYear>2010</inceptionYear>
 
-    <artifactId>testlink</artifactId>
-    <name>Jenkins TestLink Plugin</name>
-    <version>3.12-SNAPSHOT</version>
-    <packaging>hpi</packaging>
-    <description>This plug-in integrates Jenkins to TestLink</description>
+	<artifactId>testlink</artifactId>
+	<name>Jenkins TestLink Plugin</name>
+	<version>3.13-SNAPSHOT</version>
```

Comment:
The version value is changed by maven-release-plugin. When I run mvn clean release:prepare it runs the tests, does some other checks like git status and also asks for the next release version. Will fix it after merging.